packages/expo-example-notice/package.json

```diff
@@ -15,7 +15,8 @@
     "expo-splash-screen": "~0.29.22",
     "expo-status-bar": "~2.0.1",
     "react": "18.3.1",
-    "react-native": "0.76.7"
+    "react-native": "0.76.7",
+    "react-native-legal": "*"
```

Comment:
If we add local dependency here, then let's remove it from  `packages/expo-example-notice/react-native.config.js` (remove the whole file) and from the `packages/expo-example-notice/tsconfig.json` (the compilerOptions object)

Also you forgot about the `packages/expo-example-notice/metro.config.js`
```
config.resolver.nodeModulesPaths = [
  projectRoot,
  path.resolve(workspaceRoot, 'node_modules'),
  path.join(__dirname, '../with-react-native-oss-notice'), // <--- needs to be changed
];
```